### PR TITLE
latinpad ADD BOOTMAGIC_ENABLE = lite

### PR DESCRIPTION
**Pull request by latincompass**
latinpad ADD BOOTMAGIC_ENABLE = lite

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/latinpad/config.h [hunk 1]
```diff
@@ -57,3 +57,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.*/
 #define ENCODERS_PAD_B { B2, B6 }
 
 #define ENCODER_RESOLUTION 4
+/* Bootmagic Lite key configuration */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
```
Comment (zvecr) [inline keyboards/latinpad/config.h:62]: These are the defaults so not required
```suggestion
```

Comment (latincompass) [reply keyboards/latinpad/config.h]: Thanks a lot,sir

**Comment by latincompass** [comment]
I accidentally deleted the collaborator，sorry

**Comment by noroadsleft** [comment]
Thanks!

**Comment by latincompass** [comment]
> Thanks!
 Thanks a lot,sir.thanks.

**Pull request merged**